test/AccessTokenConsumer/AccessTokenConsumer.behaviour.ts

```diff
@@ -1,8 +1,8 @@
 import { ethers, waffle } from "hardhat";
```

Comment:
This file is quite big! Since there's a lot of tests repeated, just with different function signatures I feel we could create some abstraction to shrink the size of this file. Let's create an issue?

Reply:
Sure :) Issue would be great

Reply:
Nevermind, already solved ;)